Tevling/Pages/Dashboard.razor

```diff
@@ -0,0 +1,54 @@
+@page "/"
```

Comment:
You should move the redirect code from `Activities.razor.cs` to `Dashboard.razor.cs` now that this is the default page.